src/pages/gen2/start/index.mdx

```diff
@@ -0,0 +1,237 @@
+export const meta = {
+  title: 'Getting Started',
+  description: 'Learn how to get started with AWS Amplify (Gen 2)'
+};
+
```

Comment:
Recommend we add a short preview of what follows with an estimated time to complete end to end. Something like - "This *Getting Started* guide walks you through the setting up your project and personal cloud sandbox, configuring authentication and data, and deploying and hosting your app. If you are starting a new project this walk through will take you about XX minutes/hours to complete."

Reply:
Added the description and time estimate